common/screenshotter.go

```diff
@@ -61,11 +61,12 @@ func (f *ImageFormat) UnmarshalJSON(b []byte) error {
 }
 
 type screenshotter struct {
-	ctx context.Context
+	ctx       context.Context
+	persister *storage.LocalFilePersister
 }
 
-func newScreenshotter(ctx context.Context) *screenshotter {
-	return &screenshotter{ctx}
+func newScreenshotter(ctx context.Context, persister *storage.LocalFilePersister) *screenshotter {
```

Comment:
Ah, one more `persister` here. Just noticed.

Reply:
Resolved in 39d2db9dabffe683fd59362178761634f1eedd0e